### PR TITLE
Investigations Print Preview Page

### LINKED ISSUE #8583: Increase the width of investigation view details page padding
**Bug Description**

An unnecessary scrollbar appears on a standard laptop screen on the investigation details page.

Steps to Reproduce:

- [ ] Navigate to the "Patient" tab and select any card with an active consultation.
- [ ] Click on the "Investigation" tab and select the "Log Update Result" button.
- [ ] After creating a log, click on the "View" button on the card of the newly created log.
- [ ] Observe the error: a scrollbar appears unnecessarily.

**Expected Behavior**

 Adjust the width of the investigation details page for standard laptop screen sizes, ensuring that no horizontal scrolling is required.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Jacobjeevan**
## Proposed Changes

- Fixes #8583 
- Created a different print preview page for investigations, similar to prescriptions print preview (Medications tab)
- Modified Investigations view page to have no scrollbar (+ print button redirects to print preview page instead)

### Investigations tab

image

### Investigations View Page

image

image

### Investigations Print Preview Page

image

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/Investigations/ViewInvestigations.tsx [hunk 1]
```diff
@@ -58,16 +60,31 @@ export default function ViewInvestigations(props: {
...
+                  <ButtonV2
+                    onClick={() =>
+                      navigate(
+                        `/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/investigation/${investigationSession.session_external_id}`,
+                      )
+                    }
+                    ghost
+                    border
+                  >
+                    View
```
Comment (rithviknishad) [inline src/Components/Facility/Investigations/ViewInvestigations.tsx]: can we keep this in i18n

File: src/Components/Facility/Investigations/ViewInvestigations.tsx [hunk 2]
```diff
@@ -58,16 +60,31 @@ export default function ViewInvestigations(props: {
...
+                    onClick={() =>
+                      navigate(
+                        `/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/investigation/${investigationSession.session_external_id}/print`,
+                      )
+                    }
+                    ghost
+                    border
+                  >
+                    <CareIcon icon="l-print" className="text-lg" />
+                    Print
```
Comment (rithviknishad) [inline src/Components/Facility/Investigations/ViewInvestigations.tsx]: i18n

File: src/Components/Facility/Investigations/InvestigationTable.tsx [hunk 3]
```diff
@@ -127,17 +127,15 @@ export const InvestigationTable = ({
...
-                    <th
-                      key={heading}
-                      scope="col"
-                      className="px-6 py-3 text-left text-xs font-semibold uppercase tracking-wider text-secondary-800 print:text-sm print:font-normal print:tracking-normal print:text-black"
-                    >
-                      {heading}
-                    </th>
-                  ),
-                )}
+                {["Name", "Value", "Ideal"].map((heading) => (
```
Comment (rithviknishad) [inline src/Components/Facility/Investigations/InvestigationTable.tsx]: i18n

**Review by rithviknishad — COMMENTED** [review]
LGTM, just i18n

**Comment by nihal467** [comment]
@Jacobjeevan 

image

- [ ] in mobile view, the inputted data in the field is not visible 

image

- [ ] reduce the width of the value field in normal view, as its not that much required

image

- [ ] when we save the edited value, the users have been redirected to the consultation dashboard, it shouldn't be the expected behavior, keep the user in the same page itself to view the updated value

**Comment by Jacobjeevan** [comment]
>   * [x]  in mobile view, the inputted data in the field is not visible
>   * [x]  reduce the width of the value field in normal view, as its not that much required
>   * [x]  when we save the edited value, the users have been redirected to the consultation dashboard, it shouldn't be the expected behavior, keep the user in the same page itself to view the updated value

👍  @nihal467

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
image

@aparnacoronasafe can you look into the new investigation print preview, without requesting @Jacobjeevan added patient admission, IP/OP no, allergies, and bed no to the print preview of the investigation, is it ok to keep it or should we remove them and only keep the patient name and facility name in the investigation print preview

**Comment by bodhish** [comment]
~Lets add it; the prints should eventually have everything we store.~
Lets remove it @Jacobjeevan, Investigation doesn't need it

**Comment by Jacobjeevan** [comment]
@nihal467 I was working on #8275, should we do the same thing here and move print button inside?

image


image


image

**Comment by nihal467** [comment]
@Jacobjeevan yes we can do it and once the PR is ready, let me know

**Comment by Jacobjeevan** [comment]
@nihal467 Ready 👍

**Comment by nihal467** [comment]
image

@Jacobjeevan the mobile responsive bug is back, in mobile, if you type three-digit, you are not able to view it together

**Comment by Jacobjeevan** [comment]
> image
> 
> @Jacobjeevan the mobile responsive bug is back, in mobile, if you type three-digit, you are not able to view it together

@nihal467 Weird, I checked firefox and safari and it's fine for me 🤔 . But, I've adjusted the widths, take a look and see if the issue persists. 

image

**Comment by nihal467** [comment]
image

@Jacobjeevan the persist in Chrome browser, responsive view of iPhone 14 pro, can you look it

**Comment by Jacobjeevan** [comment]
> image
> 
> @Jacobjeevan the persist in Chrome browser, responsive view of iPhone 14 pro, can you look it

@nihal467 Got it 👍  Take a look now.

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**